sunkit_pyvista/plotter.py

```diff
@@ -1,10 +1,138 @@
+import functools
+
+import numpy as np
+import pyvista as pv
+
+import astropy.units as u
+from astropy.constants import R_sun
+from astropy.coordinates import SkyCoord
+
+from sunpy.map.maputils import all_corner_coords_from_map
+from sunkit_pyvista.draw import Draw
+
 
 __all__ = ['SunpyPlotter']
 
 
 class SunpyPlotter:
     """
     A plotter for 3D data.
-
     This class wraps `pyvsita.Plotter` to provide coordinate-aware plotting.
+    For now, all coordinates are converted to
+    and distance units are such that :math:`R_{sun} = 1`.
+
+    Parameters
+    ----------
+    coordinate_frame : astropy.coordinates.BaseFrame
+        Coordinate frame of the plot. The x, y, z axes of the pyvista plotter
+        will be the x, y, z axes in this coordinate system.
     """
+
+    def __init__(self, coordinate_frame=None):
+        self._plotter = pv.Plotter()
+        self.draw = Draw(coordinate_frame)
+
+    @property
+    def plotter(self):
+        """
+        `pyvista.Plotter`.
+        """
+        return self._plotter
+
+    @functools.wraps(pv.Plotter.show)
+    def show(self, *args, **kwargs):
+        """
+        Show the plot.
+        """
```

Comment:
I think there shouldn't be a docstring here, so the method can automatically inherit the docstring from `pyvista.Plotter.show`.